Vagrantfile

```diff
@@ -17,20 +17,33 @@ vmnames = nodes.times.collect { |n| "s-#{n+1}" }
 hostnames = nodes.times.collect { |n| "storageos-#{n + 1}-#{random}" }
 vm_to_host = Hash[vmnames.zip(hostnames)]
 
+memory = 2048
+cpus = 2
+extra_disk_size = 50 * 1024 # 50 GB
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "maier/alpine-3.4-x86_64"
+  config.vm.box = "generic/alpine39"
   config.vm.network "private_network", type: "dhcp"
 
+  config.vbguest.auto_update = false
+
   vmnames.each do |vmname|
     config.vm.define vmname do |node|
       node.vm.hostname = vm_to_host[vmname]
-    end
-  end
 
-  config.vm.provider "virtualbox" do |v|
-    v.memory = 2048
-    v.cpus = 2
+      node.vm.provider :virtualbox do |v|
+        v.memory = memory
+        v.cpus = cpus
+
+        #v.customize ['storagectl', :id, '--name',  'SATA Controller', '--add', 'sata',  '--controller', 'IntelAhci', '--portcount', 4]
+
+        file_to_disk = "./.vagrant/#{vmname}.vdi"
```

Comment:
the provider part was moved into the `config.vm.define` to make use of the `vmname` variable for the extra disk filename.